receiver/awscontainerinsightreceiver/internal/k8sapiserver/k8sapiserver.go

```diff
@@ -442,6 +451,35 @@ func (k *K8sAPIServer) getKubernetesBlob(pod *k8sclient.PodInfo, kubernetesBlob
 	}
 }
 
+func (k *K8sAPIServer) getHyperPodResiliencyMetrics(clusterName, timestampNs string) []pmetric.Metrics {
+	var metrics []pmetric.Metrics
+	nodeInfos := k.leaderElection.nodeClient.NodeInfos()
+	for nodeName, labels := range k.leaderElection.nodeClient.NodeToLabelsMap() {
+		if nodeInfo, ok := nodeInfos[nodeName]; ok {
+			if isHyperPodNode(nodeInfo.InstanceType) {
```

Comment:
Assuming we do go with the suggestion above to just capture the status as a string on the NodeInfo itself, probably dont need this anymore either since you can check for non-empty string.

Reply:
Agreed above.